code/game/turfs/simulated/floor/indestructible.dm

```diff
@@ -0,0 +1,82 @@
+/turf/simulated/floor/indestructible
```

Comment:
This and the indestructible walls are ugly as hell. Thing is, we can't make these walls unsimulated as they still need to process atmos. As such, this is a solution until (if ever) we port over the turf refactor.